source/includes/api/_layer.md

```diff
@@ -350,185 +524,307 @@ curl -X GET https://api.resourcewatch.org/v1/layer?includes=user
-      "description": null,
-      "layerConfig": {
-        "account": "rw",
-        "body": {
-          "maxzoom": 18,
-          "minzoom": 3,
-          "layers": [
-            {
-              "type": "mapnik",
-              "options": {
-                "sql": "SELECT * cait_2_0_country_ghg_emissions_filtered",
-                "cartocss": "",
-                "cartocss_version": "2.3.0"
-              }
-            }
-          ]
-        }
-      },
-      "legendConfig": {
-        "marks": {
-          "type": "rect",
-          "from": {
-            "data": "table"
-          }
-        }
-      },
-      "applicationConfig": {},
-      "staticImageConfig": {}
-    }
-  },
-  "meta": {
-    "status": "saved",
-    "published": true,
-    "updatedAt": "2017-01-23T16:51:42.571Z",
-    "createdAt": "2017-01-23T16:51:42.571Z"
-  }
+   "data": {
+       "id": "e5c3e7c5-19ae-4ca0-a461-71f1f67aa553",
+       "type": "layer",
+       "attributes": {
+           "slug": "total-co2-emissions-by-year",
+           "userId": "5858f37140621f11066fb2f7",
+           "application": [
+               "rw"
+           ],
+           "name": "Total CO2 emissions by year",
+           "default": false,
+           "dataset": "11de2bc1-368b-42ed-a207-aaff8ece752b",
+           "env": "production",
+           "provider": "cartodb",
+           "iso": [],
+           "description": null,
+           "layerConfig": {
+               "account": "rw",
+               "body": {
+                   "maxzoom": 18,
+                   "minzoom": 3,
+                   "layers": [
+                       {
+                           "type": "mapnik",
+                           "options": {
+                               "sql": "SELECT * cait_2_0_country_ghg_emissions_filtered",
+                               "cartocss": "",
+                               "cartocss_version": "2.3.0"
+                           }
+                       }
+                   ]
+               }
+           },
+           "legendConfig": {
+               "marks": {
+                   "type": "rect",
+                   "from": {
+                       "data": "table"
+                   }
+               }
+           },
+           "applicationConfig": {},
+           "staticImageConfig": {}
+       }
+   },
+   "meta": {
+       "status": "saved",
+       "published": true,
+       "updatedAt": "2017-01-23T16:51:42.571Z",
+       "createdAt": "2017-01-23T16:51:42.571Z"
+   }
 }
 ```
 
-### Include related entities
-
-You can load related `user` and `vocabulary` data in the same request. See [this section](#include-related-layer-entities) for more details.
+If you know the id or the `slug` of a layer, then you can access it directly. Both id and `slug` are case-sensitive.
 
+### Include related entities
 
-## Create a Layer
+You can load related `user` and `vocabulary` data in the same request. See [this section](#include-related-entities135) for more details.
```

Comment:
Looks like I jumped the gun on my previous comment, although this link doesn't work.

Reply:
If I understood correctly, that's a different thing:
- your previous comment was about having a common "includes" section that would be shared by dataset, widgets, layers, etc etc etc. While the idea is good, the fact that different resources support different includes makes me think it's implementation may be confusing to the end user, as I commented above
- This link (which is indeed broken, and is putting up a fight) points to the "includes" from the same layer resource - it basically means your get single layer request supports the same includes as you get multiple layers request.

I'll keep you up to date on the link bug

Reply:
update on link bug: this seems to be an issue with slate, the tool we use to build the html from the markdown files. The only solution I could find was to slightly change the titles of the sections, so that they don't repeat.